### DISCUSSION TITLE
Erkennung ob 230V - Adapterkabel angeschlossen ist

### DISCUSSION THREAD

**Discussion by masterkey2000** [MK-2001/ioBroker.go-e#132] [Q&A] [answered]
Hallo,

ich programmiere mit gerade eine PV-Überschussladung. Ich würde diese Überschussladung gerne nur erlauben wenn der 230V Adapterstecker angeschlossen ist. In der App des go-e chargers wird mir angezeigt, ob der Adaperstecker (1phasig) angeschlossen ist oder nicht).
Gibt es im go-e charger Adapter einen Datenpunkt der disen Status ausgibt?

**Comment by MK-2001** [comment, marked as answer]
HI, 

go-e.0.energy.phase2.postContactorActive bzw. passe 1 & 3entsprechend.
Das ist, ob das Auto mit mehr als einer Phase lädt.

go-e.0.energy.phase2.preContactorActive 
Das ist ob am Adapter eine Phase angeschlossen ist.

Gruß

**Reply by masterkey2000** [reply]
Super! Danke. Damit kann ich was anfangen. 

Was ist mit dem Wert go-e.0.phases?
Mit ist aufgefallen dass auf auf 8 steht wenn der Schuko-Stecker Adapter dran hängt und auf 56 wenn er direkt am Starkstrom hängt. Was haben die Werte zu bedeuten?

**Reply by MK-2001** [reply]
Das ist das kombinierte Phasen Feld:
Eine Beschreibung gibt es hier: https://github.com/goecharger/go-eCharger-API-v1/blob/master/go-eCharger%20API%20v1%20DE.md#parameter

=> pha 

Gruß
